.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  ROCKS_USERNAME: ${{ secrets.ROCKS_USERNAME }}
+  ROCKS_PASSWORD: ${{ secrets.ROCKS_PASSWORD }}
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rosik/setup-tarantool@v1
+        with: {tarantool-version: '1.10'}
+
+      - name: Upload rockspec
+        run: |
+          TAG=${GITHUB_REF##*/}
+          ./release.sh $TAG
+          ./upload.sh release/luarapidxml-${TAG}-1.rockspec
+          ./upload.sh release/luarapidxml-${TAG}-1.src.rock
```

Comment:
For future, seems we need separate action that accepts rock name and credentials and push rockspec.